docs/src/main/asciidoc/appcds.adoc

```diff
@@ -0,0 +1,95 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
+////
+= AppCDS
+include::_attributes.adoc[]
+:categories: core, cloud
+:summary: This reference guide explains how to enable AppCDS with Quarkus.
+
+This reference guide explains how to enable Application Class Data Sharing in your Quarkus applications.
+
+== What is Application Class Data Sharing (AppCDS)?
+
+link:https://docs.oracle.com/en/java/javase/17/vm/class-data-sharing.html[Application Class Data Sharing] is a JVM feature that helps reduce the startup time and memory footprint of a JVM application.
```

Comment:
Shouldn't this point to `https://docs.oracle.com/en/java/javase/17/docs/specs/man/java.html#application-class-data-sharing` ? There is CDS (Class Data Sharing) and *Application* Class Data Sharing.

Reply:
Sure, why not

Reply:
Sure, fixed